kasa/aestransport.py

```diff
@@ -184,8 +185,24 @@ async def send_secure_passthrough(self, request: str) -> Dict[str, Any]:
             assert self._encryption_session is not None
 
         raw_response: str = resp_dict["result"]["response"]
-        response = self._encryption_session.decrypt(raw_response.encode())
-        return json_loads(response)  # type: ignore[return-value]
+
+        try:
+            response = self._encryption_session.decrypt(raw_response.encode())
+            ret_val = json_loads(response)
+        except Exception as ex:
+            try:
+                ret_val = json_loads(raw_response)
+                _LOGGER.debug(
+                    "%s Secure passthrough response was received unencrypted!",
+                    self._host,
+                )
+            except Exception:
+                raise SmartDeviceException(
+                    f"Unable to decrypt response from {self._host}, "
+                    + f"error: {ex}, response: {raw_response}",
+                    ex,
```

Comment:
Hmm, I'm not sure if ex should be passed here?

Reply:
When we pass ex in we get more information in the HA logs, i.e. the exception type.  Otherwise we just get the message which can sometimes be ambiguous.

Reply:
I meant that this is now doing `raise SmartDeviceException("msg", ex) from ex`, i.e., the information is passed twice. But I'm okay with that, it's not a big issue.